lib/yardstick/v2_client/booking_request.rb

```diff
@@ -20,6 +20,7 @@ def self.process_response(resp, extras = {})
         end
         attrs[:local_start_at] = DateTime.parse(attrs[:local_start_at]) if attrs[:local_start_at].present?
         attrs[:local_end_at] = DateTime.parse(attrs[:local_end_at]) if attrs[:local_end_at].present?
+        attrs[:earliest_date_available] = DateTime.parse(attrs[:earliest_date_available]) if attrs[:earliest_date_available].present?
```

Comment:
Line is too long. [133/80]<br>Do not use `DateTime.parse` without zone. Use one of `Time.zone.parse`, `DateTime.parse.current`, `DateTime.parse.in_time_zone`, `DateTime.parse.utc`, `DateTime.parse.getlocal`, `DateTime.parse.iso8601`, `DateTime.parse.jisx0301`, `DateTime.parse.rfc3339`, `DateTime.parse.to_i`, `DateTime.parse.to_f` instead.